UI/window-basic-adv-audio.cpp

```diff
@@ -1,134 +1,38 @@
-	usePercent->setText("%");
-	connect(usePercent, SIGNAL(toggled(bool)), this,
-		SLOT(SetVolumeType(bool)));
+	ui->setupUi(this);
 
 	VolumeType volType = (VolumeType)config_get_int(
 		GetGlobalConfig(), "BasicWindow", "AdvAudioVolumeType");
 
 	if (volType == VolumeType::Percent)
-		usePercent->setChecked(true);
-
-	QHBoxLayout *volLayout = new QHBoxLayout();
-	volLayout->setContentsMargins(0, 0, 0, 0);
-	volLayout->addWidget(volLabel);
-	volLayout->addWidget(usePercent);
-	volLayout->addStretch();
-
-	int idx = 0;
-	mainLayout = new QGridLayout;
-	mainLayout->setContentsMargins(0, 0, 0, 0);
-	label = new QLabel("");
-	mainLayout->addWidget(label, 0, idx++);
-	label = new QLabel(QTStr("Basic.AdvAudio.Name"));
-	label->setStyleSheet("font-weight: bold;");
-	mainLayout->addWidget(label, 0, idx++);
-	label = new QLabel(QTStr("Basic.Stats.Status"));
-	label->setStyleSheet("font-weight: bold;");
-	mainLayout->addWidget(label, 0, idx++);
-	mainLayout->addLayout(volLayout, 0, idx++);
-	label = new QLabel(QTStr("Basic.AdvAudio.Mono"));
-	label->setStyleSheet("font-weight: bold;");
-	mainLayout->addWidget(label, 0, idx++);
-	label = new QLabel(QTStr("Basic.AdvAudio.Balance"));
-	label->setStyleSheet("font-weight: bold;");
-	mainLayout->addWidget(label, 0, idx++);
-	label = new QLabel(QTStr("Basic.AdvAudio.SyncOffset"));
-	label->setStyleSheet("font-weight: bold;");
-	mainLayout->addWidget(label, 0, idx++);
-	if (obs_audio_monitoring_available()) {
-		label = new QLabel(QTStr("Basic.AdvAudio.Monitoring"));
-		label->setStyleSheet("font-weight: bold;");
-		mainLayout->addWidget(label, 0, idx++);
-	}
-	label = new QLabel(QTStr("Basic.AdvAudio.AudioTracks"));
-	label->setStyleSheet("font-weight: bold;");
-	mainLayout->addWidget(label, 0, idx++);
-
-	controlArea = new QWidget;
-	controlArea->setLayout(mainLayout);
-	controlArea->setSizePolicy(QSizePolicy::Preferred,
-				   QSizePolicy::Preferred);
-
-	vlayout = new QVBoxLayout;
-	vlayout->addWidget(controlArea);
-	//vlayout->setAlignment(controlArea, Qt::AlignTop);
-	widget = new QWidget;
-	widget->setLayout(vlayout);
-	widget->setSizePolicy(QSizePolicy::Preferred, QSizePolicy::Maximum);
-
-	scrollArea = new QScrollArea;
-	scrollArea->setWidget(widget);
-	scrollArea->setWidgetResizable(true);
-
-	QPushButton *closeButton = new QPushButton(QTStr("Close"));
-
-	activeOnly = new QCheckBox();
-	activeOnly->setChecked(!showInactive);
-	activeOnly->setText(QTStr("Basic.AdvAudio.ActiveOnly"));
-
-	QHBoxLayout *buttonLayout = new QHBoxLayout;
-	buttonLayout->addWidget(activeOnly);
-	buttonLayout->addStretch();
-	buttonLayout->addWidget(closeButton);
-
-	vlayout = new QVBoxLayout;
-	vlayout->setContentsMargins(11, 11, 11, 11);
-	vlayout->addWidget(scrollArea);
-	vlayout->addLayout(buttonLayout);
-	setLayout(vlayout);
-
-	connect(activeOnly, SIGNAL(clicked(bool)), this,
-		SLOT(ActiveOnlyChanged(bool)));
-
-	connect(closeButton, &QPushButton::clicked, [this]() { close(); });
+		on_usePercent_toggled(true);
```

Comment:
```suggestion
		ui->usePercent->blockSignals(true);
		ui->usePercent->setChecked(true);
		ui->usePercent->blockSignals(false);
	}
```

Reply:
Fixed. I didn't add the block signals here because we still need to have to toggle function called.